frappe/desk/doctype/system_console/system_console.js

```diff
@@ -96,7 +97,7 @@ frappe.ui.form.on('System Console', {
 					<th width='10%'>Time</ht>
 					<th width='10%'>State</ht>
 					<th width='60%'>Info</ht>
-					<th width='10%'>Progress</ht>
+					<th width='10%'>Progress/Wait Event</ht>
```

Comment:
```suggestion
					<th width='25%'>Progress / Wait Event</ht>
```

And make ID 5% too? Looks better this way.

Reply:
It should be 5% and 15% right?